pom.xml

```diff
@@ -128,12 +128,33 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.codehaus.mojo</groupId>
```

Comment:
So for example this would tell you something like ZoneId is not available in JDK 1.7 even though you are developing using JDK 1.8? How come the compiler plugin wouldn't fail if your target JDK was 1.7?